bin/posydon-run-pipeline

```diff
@@ -482,9 +482,9 @@ def copy_ini_file(grid_type, rerun_metallicity, rerun_type, destination, cluster
     elif rerun_type == 'initial_He':
         replace_text = "fix_iniHe_pre_rerun5-0e736a853c3c2e522b76299a54aea8f5eea15d08"
     elif rerun_type == "timestep_MTlimit":
-        replace_text = "seth_HMS-HMS_rerun_accretion_fix-f74d4ed021ae67aaa7b89e77da33041550a7d71a"
+        replace_text = "seth_HMS_HMS_rerun_accretion_fix-f74d4ed021ae67aaa7b89e77da33041550a7d71a"
     elif rerun_type == "egrav_Pgas_dXdt_terms":
-        replace_text = "seth_CO-HeMS_rerun_fix-ecae5eecfc3cecfb20f50b673d0f575bc5a75998"
+        replace_text = "seth_CO_HeMS_rerun_fix-ecae5eecfc3cecfb20f50b673d0f575bc5a75998"
```

Comment:
It might have been better to rename the branches, to keep our convention of using a `-` between star types in the grid type names.

Reply:
I've renamed the branches to just use the rerun type names for now instead, as there's no real need for me to specify the grids anymore in their names since the implementations are generalized.

As a note though, in `posydon-setup-grid`, the code checks for the branch name and SHA using

```
branch = gitcommit.split('-')[0]
githash = gitcommit.split('-')[1]
```

Should this be changed to e.g.,

```
branch = gitcommit.rsplit('-', 1)[0]
githash = gitcommit.rsplit('-', 1)[-1]
```

This could allow for '-' characters in the branch name so it can be recorded consistently in this code and on github.

Reply:
I think it would be good to make this change to allow a "-" in the branch name.
I don't see a reason to not just include this change here in this PR-